common/component/redis/v9client.go

```diff
@@ -395,9 +400,10 @@ func newV9Client(s *Settings) RedisClient {
 				InsecureSkipVerify: s.EnableTLS,
 			}
 		}
-
+		client := v9.NewClusterClient(options)
+		go refreshTokenRoutineForRedis(context.Background(), ClientFromV9Client(client), properties)
```

Comment:
Is it the case that we _always_ want to be doing this refresh? Should this not be an option?

Reply:
ya it seems the connection might terminate so we must be doing it continuously in some interval